telemetry/src/dev.rs

```diff
@@ -30,7 +25,7 @@ pub async fn start(
             //.append(true)
             .create(true)
             .truncate(true)
-            .open(telemetry_file)
+            .open(config.file)
```

Comment:
```suggestion
            .open(config.out)
```

`file` seems ambiguous?

Reply:
Hm, let me change to `out_file`. Will also change it in the user config.